apps/web/composables/useBlockManager/useBlockManager.ts

```diff
@@ -7,6 +9,8 @@ const blockHasData = (block: Block): boolean => !isEmptyBlock(block);
 
 export function useBlockManager() {
   const { data } = useHomepage();
+  const { isEditing, isEditingEnabled } = useEditor();
+  const { initialBlocks } = useHomepage();
```

Comment:
```suggestion
  const { data, initialBlocks } = useHomepage();
  const { isEditing, isEditingEnabled } = useEditor();
```
Everything else looks good to me.